homework3/task2/main.c

```diff
@@ -0,0 +1,121 @@
+#include "../../lib/functions.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+int compare(const int number1, const int number2)
+{
+    if (number1 == number2) return 0;
+    else if (number1 < number2) return -1;
+    else return 1;
```

Comment:
По стайлгайду так:
```suggestion
    if (number1 == number2) 
    {
        return 0;
    }
    else if (number1 < number2) 
    {
        return -1;
    }
    return 1;
```
Так проще отлаживаться, потому что точку останова в Си можно поставить на строчку, а не на конкретный оператор.